### PR TITLE
fix(vim): support syntax command and update parser

### PR DESCRIPTION
**Pull request by vigoux**

### CODE REVIEW

**Comment by theHamsta** [comment]
Can you rebase this?

**Review by theHamsta — APPROVED** [review]

**Comment by vigoux** [comment]
Should be good now

File: lockfile.json [hunk 1]
```diff
@@ -213,7 +213,7 @@
     "revision": "8f6b1f357d1231c420404b5f7a368a73c25adfa2"
   },
   "vim": {
-    "revision": "59595443fb486449f42db839934117221358a85f"
+    "revision": "7d883fb13bf4f9038522c9d8b3197b821498ab1d"
```
Comment (stsewd) [inline lockfile.json:216]: I think CI is failing because this commit doesn't exist. https://github.com/vigoux/tree-sitter-viml/commits/master

**Comment by theHamsta** [comment]
@vigoux

**Comment by stsewd** [comment]
Merged via https://github.com/nvim-treesitter/nvim-treesitter/pull/1878

**Pull request closed without merging**